AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/src/Mutations.dfy

```diff
@@ -23,6 +23,10 @@ module {:options "/functionSyntax:4" } Mutations {
   import MutationErrorRefinement
   import KmsUtils
 
+  datatype InitialHVVerificationState =
```

Comment:
There could be a better name then this. I will come up with something. Any suggestions?